src/PIL/ImageDraw.py

```diff
@@ -261,24 +261,95 @@ def _multiline_split(self, text):
 
         return text.split(split_character)
 
-    def text(self, xy, text, fill=None, font=None, anchor=None, *args, **kwargs):
+    def text(
+        self,
+        xy,
+        text,
+        fill=None,
+        font=None,
+        anchor=None,
+        spacing=4,
+        align="left",
+        direction=None,
+        features=None,
+        language=None,
+        strokeWidth=0,
+        strokeFill=None,
```

Comment:
Any particular reason for stroke width and fill params to be in this order (and in `multiline_text`)?

If not, how about alphabetical order?

Reply:
The only reason would be that stroke fill has no meaning without stroke width, making stroke fill optional compared to stroke width - in the situation where a user wanted to stroke with the same fill as the text, they would specify width, but leave fill as the default.

If you think it's better to easily remember the arguments as being in alphabetical order though, then I can make that change.